data/blog/electron-14-0.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Electron 14.0.0
+author:
+- electron
+date: '2021-08-31'
+---
+
+Electron 14.0.0 has been released! It includes upgrades to Chromium `93` and V8 `9.3`. We've added several API updates, bug fixes, and general improvements. Read below for more details!
+
+---
+
+The Electron team is excited to announce the release of Electron 14.0.0! You can install it with npm via `npm install electron@latest` or download it from our [releases website](https://electronjs.org/releases/stable). Continue reading for details about this release, and please share any feedback you have!
+
+## Notable Changes
+
+### Stack Changes
+
+* Chromium `93`
+    * [New in Chrome 91](https://developer.chrome.com/blog/new-in-chrome-91/)
+    * [New in Chrome 90](https://developer.chrome.com/blog/new-in-chrome-90/)
+* Node.js `14.17.0`
+    * [Node 14.17.0 blog post](https://nodejs.org/en/blog/release/v14.17.0/)
+* V8 `9.3`
+    * [V8 9.3 blog post](https://v8.dev/blog/v8-release-93)
+    * [V8 9.2 blog post](https://v8.dev/blog/v8-release-92)
+
+### Highlight Features
+
+* TODO: Add highlight features
+
+See the [14.0.0 release notes](https://github.com/electron/electron/releases/tag/v14.0.0) for a full list of new features and changes.
+
+## Breaking Changes
+
+* TODO: Add breaking changes
+
+More information about these and future changes can be found on the [Planned Breaking Changes](https://github.com/electron/electron/blob/master/docs/breaking-changes.md) page.
+
+## API Changes
+
+* TODO: add any API changes
+
+### Removed/Deprecated Changes
+
+The following APIs have been removed or are now deprecated:
+
+* TODO
+
+## End of Support for 11.x.y
+
+Electron 11.x.y has reached end-of-support as per the project's [support policy](https://electronjs.org/docs/tutorial/support#supported-versions). Developers and applications are encouraged to upgrade to a newer version of Electron.
+
+## What's Next
+
+In the short term, you can expect the team to continue to focus on keeping up with the development of the major components that make up Electron, including Chromium, Node, and V8. Although we are careful not to make promises about release dates, our plan is release new major versions of Electron with new versions of those components approximately quarterly. The [tentative 14.0.0 schedule](https://electronjs.org/docs/tutorial/electron-timelines) maps out key dates in the Electron 14.0 development life cycle. Also, [see our versioning document](https://electronjs.org/docs/tutorial/electron-versioning) for more detailed information about versioning in Electron.
```

Comment:
Seems like the info about versioning should probably be put into the `New Electron Cadence` section?